react/README.md

```diff
@@ -386,7 +386,7 @@
 
     // good
     <Foo
-      ref={(ref) => { this.myRef = ref; }}
+      ref={ref => this.myRef = ref}
```

Comment:
It's valid in JavaScript. It's not valid according to this styleguide (read the main guide).